task.go

```diff
@@ -1,5 +1,10 @@
 package onfleet
 
+type PaginatedTasks struct {
```

Comment:
for consistency with rest of file I suggest naming this `TasksPaginated` instead of `PaginatedTasks`